README.md

```diff
@@ -20,16 +20,7 @@ Goravel Installer is a command-line tool that helps you to install the Goravel f
 
 ```bash
 # Install the latest version of the goravel installer
-go install github.com/goravel/installer@latest
-
-# You can rename the executable file
-# Linux / MacOS
-mv "$GOBIN/installer" "$GOBIN/goravel"
-
-# Windows
-move "%GOBIN%\installer.exe" "%GOBIN%\goravel.exe"
-# Windows Powershell
-move "$Env:gopath\bin\installer.exe" "$Env:gopath\bin\goravel.exe"
+go install github.com/goravel/installer/goravel@latest
```

Comment:
returns error?

<img width="698" alt="image" src="https://github.com/user-attachments/assets/8a80cbd6-a138-4a81-97c9-ae0fa6cde235" />

Reply:
Test `go install github.com/goravel/installer/goravel@master` before you have new release.

Reply:
The same:

<img width="706" alt="image" src="https://github.com/user-attachments/assets/dea9d3c7-93a9-4e97-bdaa-f0c919f52e8e" />

Reply:
you have merged?